### PR TITLE
config: adding builder db host param (PROJQUAY-8184)

### PR DESCRIPTION
**Pull request by bcaton85**

### CODE REVIEW

**Review by jonathankingfc — APPROVED** [review]
LGTM

File: data/database.py [hunk 1]
```diff
@@ -427,11 +427,16 @@ def _db_from_url(
     allow_pooling=True,
     allow_retry=True,
     is_read_replica=False,
+    builder_host_override=None,
 ):
     parsed_url = make_url(url)
 
     if parsed_url.host:
         db_kwargs["host"] = parsed_url.host
+        service_name = os.getenv("SUPERVISOR_PROCESS_NAME")
```
Comment (ibazulic) [inline data/database.py:436]: How is this `SUPERVISOR_PROCESS_NAME` set?

Comment (bcaton85) [reply data/database.py:436]: seems to be set by [supervisord](http://supervisord.org/subprocess.html#subprocess-environment)

Comment (ibazulic) [reply data/database.py:436]: Ah, gotcha, that makes sense :-)

**Review by ibazulic — APPROVED** [review]

**Pull request merged**